public/app/features/dashboard/components/DashboardSettings/DashboardSettings.tsx

```diff
@@ -142,6 +151,7 @@ export class DashboardSettings extends PureComponent<Props> {
     const currentPage = pages.find((page) => page.id === editview) ?? pages[0];
     const canSaveAs = contextSrv.hasEditPermissionInFolders;
     const canSave = dashboard.meta.canSave;
+    const styles = getStyles(config.theme);
```

Comment:
Not 100% but i think using the config.theme is maybe not the best idea. Since the theme in the react tree is provided via context it should be retrieved in a such way making the component independent from global variables.

As this component is a class onne, you can use [withTheme](https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#using-withtheme-higher-order-component-hoc) HOC or make it function componet and use [useStyles](https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook).

Reply:
think this is fine here, we use config.theme a lot in the main app components, we should move away from it but to fix this bug here I thbnk itä's fine, and we do not have to use withTheme which is messy

Reply:
Thanks for the feedback Dominik. When I rip out all the dashboard settings sass in favour of emotion I'll definitely introduce `withTheme` or `useStyles`.

Reply:
Thanks @jackw . In case of this PR it does not really matter.